Source/ProcessingTest/Presenters/ProcessingPresenter.cpp

```diff
@@ -116,8 +116,18 @@ void ProcessingPresenter::Show()
     IAllocator* poolAllocator = new RawPoolAllocator(150 * 1024 * 1024);
 
     OC_LOG_INFO("Loading image");
-    provider->Load(_currentFilePath, FileFormat::DNG, *_image.get(), *poolAllocator);
-    //    provider->Load("M11-1526.VB.mlv", FileFormat::MLV, *_image.get(), *poolAllocator);
+    std::string extension = _currentFilePath.substr(_currentFilePath.find_last_of(".") + 1);
+    std::transform(extension.begin(), extension.end(), extension.begin(), ::tolower);
+     
+    if(extension == "dng")
+    {
+        provider->Load(_currentFilePath, FileFormat::DNG, *_image.get(), *poolAllocator);

```

Comment:
Better to assign to format variable, than to call the method, which makes the code simpler:

```
    if(extension == "mlv")
    {
        format = FileFormat::MLV;
    }
```

Reply:
Corrected in new commit ,